cypress/integration/sample_spec.ts

```diff
@@ -22,6 +22,21 @@ describe("Home Page", () => {
   });
 });
 
+describe("Search functionality", () => {
+  beforeEach(() => {
+    setupServerMocks()
+  })
+
+  it("checks that search updates the url correctly ", () => {
+    cy.visit("/");
+    cy
+      .get(`[name="sound-search"]`)
```

Comment:
This should be query selecting by `data-e2e-id` attr, which we need to add to the input element

Reply:
Ah, okay. I'll add it!